coopdigital/coop-frontend
# React storybook

- Removed all storybook and react dependencies from the root
- Created new `react-ui` package
- Set react-ui package to be ignored by lerna
- Moved storybook, react dependencies and config to the react-ui package
- Updated configs for eslint and jest to remove references to react 
- Removed react, storybook stories and jest tests from `component-template` and `component-blockquote` packages.
- test npm run build for linting, tests, and build of packages to ensure no breaking changes were introduced.

To test:
- pull the repo on this branch `react-storybook`
- run `npm ci` at root
- cd to `packages/react-ui`
- run `npm install` inside react-ui package
- run `npm run storybook` inside react-ui package


This should spin up storybook in local development on port :6006
